test/AlgebraicGeometry/Schemes/elliptic_surface.jl

```diff
@@ -146,3 +146,42 @@ end
   g6,_ = two_neighbor_step(X2, fibers_in_X2[6])  # the non-torsion case
 end
 =#
+
+@testset "translation on elliptic surfaces" begin
+  P, t = GF(113)[:t]
+  kt = fraction_field(P)
+
+  R, (x, y) = kt[:x, :y]
+
+  f = y^2 - (x^3 + (37*t^8 + 56*t^7 + 102*t^6 + 97*t^5 + 21*t^4 + 14*t^3 + 24*t^2 + 23*t + 59)*x + 18*t^12 + 111*t^11 + 17*t^10 + 25*t^9 + 108*t^8 + 91*t^7 + 90*t^6 + 21*t^5 + 68*t^4 + 8*t^3 + 92*t^2 + 66*t + 31)
+
+  E = elliptic_curve(f, x, y)
+
+  P = E([(10*t^6 + 23*t^5 + 94*t^4 + 32*t^3 + t^2 + 40*t + 52)//(t^2 + 77*t + 98), (22*t^9 + 40*t^8 + 63*t^7 + 42*t^6 + 34*t^5 + 48*t^4 + 72*t^3 + 92*t^2 + 85*t + 91)//(t^3 + 59*t^2 + 68*t + 44)])
+
+  X = elliptic_surface(E, 2, [P])
+
+  set_verbose_level(:EllipticSurface, 5)
+
+  D_P = Oscar._section(X, P)
+
+  II = first(components(D_P));
+
+  trans = Oscar.translation_morphism(X, P; divisor=D_P)
+
+  JJ = Oscar._pushforward_section(trans, P; divisor=D_P)
+
+  IIX = first(components(Oscar._section(X, 2*P)));
+  # We have little chance to get through with the computations on all charts. 
+  # But it suffices to compare the result on the weierstrass charts.
+  weier = weierstrass_chart_on_minimal_model(X)
+  @test IIX(weier) == JJ(weier)
+
+  lat = algebraic_lattice(X)
+  A = [intersect(a, b) for a in lat, b in lat]
+
+  ll = Oscar._pushforward_lattice_along_isomorphism(trans)
+  B = [intersect(a, b) for a in ll, b in ll]
+  @test A == B
```

Comment:
I added some more expensive tests here. Maybe we should disable them again, but leave them here as comment for later reference.